doc/array.rst

```diff
@@ -339,6 +339,15 @@ Constructing :class:`GPUArray` Instances
     Return the :class:`GPUArray` ``[a[indices[0]], ..., a[indices[n]]]``.
     For the moment, *a* must be a type that can be bound to a texture.
 
+.. function:: concatenate(arrays, axis=0, allocator=None)
+
+   Join a sequence of arrays along an existing axis.
+
+.. function:: stack(arrays, axis=0, allocator=None)
+
+   Joing a sequeunce of arrays along a new axis
```

Comment:
- Typo
- Period at the end.

Reply:
Thanks for finding the typos. The typo has been fixed in my latest commit for array.rst